scss/book-list-detail.scss

```diff
@@ -0,0 +1,97 @@
+.body{
+  font-family: 'Roboto', sans-serif;
+}
+
+.master-container{
+  display: flex;
+  height: 768px;
+  width: 1366px;
+  background-color: $wild-sand;
+  justify-content: center;
+  align-items: center;
+
+  .navbar-container{
+      position: fixed;
```

Comment:
Have tried applying this rule to `.navbar`? Maybe this class (and the container element) is not necessary